js/projects.js

```diff
@@ -0,0 +1,191 @@
+//  Array storing all the projects
+const projects = [
+  {
+    name: 'Tonic',
+    description:
+        ' A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: './assets/tonic-nature.png',
+    technologies: ['html', 'css', 'javascript'],
+    tags: ['CANOPY', 'Bac End Dev', '2015'],
+    link_live: 'https://kensteph.github.io/Portfolio/',
+    link_source: 'https://github.com/kensteph/Portfolio',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description:
+        ' Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    image: './assets/multi-post-stories-2.png',
+    technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
+    tags: ['CANOPY', 'Bac End Dev', '2015'],
+    link_live: 'https://kensteph.github.io/Portfolio/',
+    link_source: 'https://github.com/kensteph/Portfolio',
+  },
+  {
+    name: 'Facebook 360',
+    description:
+        "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+    image: './assets/tonic.png',
+    technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
+    tags: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    link_live: 'https://kensteph.github.io/Portfolio/',
+    link_source: 'https://github.com/kensteph/Portfolio',
+  },
+  {
+    name: 'Uber Navigation',
+    description:
+        'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    image: './assets/multi-post-stories.png',
+    technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
+    tags: ['Uber', 'Lead Developer', '2015'],
+    link_live: 'https://kensteph.github.io/Portfolio/',
+    link_source: 'https://github.com/kensteph/Portfolio',
+  },
+];
+
+// Create li children
+const createLiChildren = (ul, project, section) => {
+  if (section === 'tag') {
+    project.tags.forEach((tag, i) => {
+      if (i === 1 || i === 2) {
+        // Create li
+        const li = document.createElement('li');
+        li.innerHTML = "<div class='bullet'></div>";
+        // Add li to ul
+        ul.appendChild(li);
+      }
+      // Create li
+      const li = document.createElement('li');
+      li.textContent = tag;
+      // Add li to ul
+      ul.appendChild(li);
+    });
+  } else {
+    project.technologies.forEach((tag) => {
+      // Create li
+      const li = document.createElement('li');
+      li.textContent = tag;
+      // Add li to ul
+      ul.appendChild(li);
+    });
+  }
+
+  return ul;
+};
+  //  Get the works section
+const workSection = document.querySelector('#works');
+
+//  Dynamicly create the cards
+projects.forEach((project, projectPos) => {
+  // console.log(project);
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻